<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plots Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Constructing Plots</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Output</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QGridLayout" name="EditorLayout"/>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="ExportData_PB">
             <property name="text">
              <string>Export data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ClearData_PB">
             <property name="text">
              <string>Clear data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Exit_PB">
             <property name="text">
              <string>Exit_PB</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string> </string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="3">
            <widget class="QLineEdit" name="Z_LE"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="X_LE"/>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="Y_Pixels_LE"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>pixels</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="Y_Width_LE"/>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>origin</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="X_Width_LE"/>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="Z_Width_LE"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>width</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="Z_Pixels_LE"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="X_Pixels_LE"/>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="Y_LE"/>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLineEdit" name="PlotName_LE"/>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="ColorBy_CB">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>material</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cell</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="PlotId_LE">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>plot_id</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QComboBox" name="Basis_CB">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>xy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>xz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>yz</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>color by</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_1">
             <property name="text">
              <string>basis</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCheckBox" name="AddPlotId_CB">
             <property name="text">
              <string>Add id to plot name</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="plot2D_RB">
             <property name="text">
              <string>2D</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="plot3D_RB">
             <property name="text">
              <string>3D</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string> </string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QPushButton" name="CreatePlot_PB">
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>120</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>PlotName_LE</tabstop>
  <tabstop>PlotId_LE</tabstop>
  <tabstop>ColorBy_CB</tabstop>
  <tabstop>Basis_CB</tabstop>
  <tabstop>plot2D_RB</tabstop>
  <tabstop>plot3D_RB</tabstop>
  <tabstop>AddPlotId_CB</tabstop>
  <tabstop>X_LE</tabstop>
  <tabstop>Y_LE</tabstop>
  <tabstop>Z_LE</tabstop>
  <tabstop>X_Width_LE</tabstop>
  <tabstop>Y_Width_LE</tabstop>
  <tabstop>Z_Width_LE</tabstop>
  <tabstop>X_Pixels_LE</tabstop>
  <tabstop>Y_Pixels_LE</tabstop>
  <tabstop>Z_Pixels_LE</tabstop>
  <tabstop>CreatePlot_PB</tabstop>
  <tabstop>ExportData_PB</tabstop>
  <tabstop>ClearData_PB</tabstop>
  <tabstop>Exit_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
